app/routes/account/renew.py

```diff
@@ -0,0 +1,67 @@
+import pexpect
+
+import ocflib.account.validators as validators
+
+from fastapi import HTTPException
+from pydantic import BaseModel, Field
+
+from routes import router
+
+
+class ExpiredPasswordInput(BaseModel):
+    username: str = Field(
+        min_length=3,
+        max_length=16,
+    )
+    old_password: str = Field(
+        min_length=5,
+        max_length=256,
+    )
+    new_password: str = Field(
+        min_length=12,
+        max_length=256,
+    )
+
+
+class ExpiredPasswordOutput(BaseModel):
+    output: str
+    error: str
+
+
+@router.post("/account/renew", tags=["account"], response_model=ExpiredPasswordOutput)
+def reset_password(data: ExpiredPasswordInput):
```

Comment:
ideally this function name would be `renew_password` or something similar since this is what FastAPI uses for the endpoint title and it could be confused with changing the user's password.

Reply:
and probably also the file name